inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/extract/MongoExtractNode.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.protocol.node.extract;
+
+import com.google.common.base.Preconditions;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonCreator;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonInclude.Include;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.inlong.sort.protocol.FieldInfo;
+import org.apache.inlong.sort.protocol.node.ExtractNode;
+import org.apache.inlong.sort.protocol.transformation.WatermarkField;
+
+public class MongoExtractNode extends ExtractNode implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @JsonInclude(Include.NON_NULL)
+    @JsonProperty("primaryKey")
+    private String primaryKey;
+    @JsonProperty("hosts")
+    private String hosts;
+    @JsonProperty("username")
+    private String username;
+    @JsonProperty("password")
+    private String password;
+    @JsonProperty("database")
+    private String database;
+    @JsonProperty("collection")
+    private String collection;
+
+    @JsonCreator
+    public MongoExtractNode(@JsonProperty("id") String id,
+        @JsonProperty("name") String name,
+        @JsonProperty("fields") List<FieldInfo> fields,
+        @Nullable @JsonProperty("watermarkField") WatermarkField waterMarkField,
+        @JsonProperty("properties") Map<String, String> properties,
+        @JsonProperty("primaryKey") String primaryKey,
+        @JsonProperty("tableNames") @Nonnull String collection,
```

Comment:
`@JsonProperty("collection")` and `@JsonProperty("tableNames")` is different. So serialization and deserialization will have some problem. Add a UT for serialization and deserialization test in inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/node/extract

Reply:
will do, thanks